src/site/README.md

```diff
@@ -307,3 +307,39 @@ The following query looks up the `entityId` for one page of the `entityBundle` (
 ```
 
 </details>
+
+## Liquid Template Unit Testing Framework
+
+The Liquid Template Unit Testing Framework was created to replace Cypress, which is a slow and heavy framework and is overkill for testing logic in `liquid` templates.
+
```

Comment:
Cypress tests do not give us control over the test data, which is crtitical for these tests.

Reply:
Can we elaborate on "The Liquid Template Unit Testing Framework was created to replace Cypress" is this a longterm update plan on the site? Last I heard some teams are still trying to migrate from Nightwatch to Cypress

Also, should we be adding this type of documentation to:

- https://department-of-veterans-affairs.github.io/va.gov-team/
- https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/common-tasks/test

Rather than in the repo like this?

Reply:
I'll add documentation to `https://department-of-veterans-affairs.github.io/va.gov-team/` in a fast-follow pr.

Reply:
This still reads like Cypress is being completely killed off, is that the case? If so I think that needs to be communicated to teams currently adding Cypress tests

Reply:
I updated that paragraph to:

```
The Liquid Template Unit Testing Framework was created to replace Cypress for unit testing the logic in `liquid` templates because Cypress is slow and heavy and is overkill for this purpose. Please note: End-to-end (e2e) tests on VA.gov use Cypress; Cypress has not been replaced by this tool.
```